### ISSUE TITLE
codegen fails due to missing Cargo.toml package.version field

### ISSUE THREAD

**Issue by jayvdb** [taiki-e/install-action#298]
This occurs on latest stable

```
install-action> ./tools/manifest.sh osv-scanner
error: failed to load manifest for workspace member `/home/jayvdb/actions/install-action/tools/codegen`

Caused by:
  failed to parse manifest at `/home/jayvdb/actions/install-action/tools/codegen/Cargo.toml`

Caused by:
  TOML parse error at line 1, column 1
    |
  1 | [package]
    | ^^^^^^^^^
  missing field `version`
```

**Comment by NobodyXu** [comment]
Perhaps setting `publish = false` will fix this?

**Comment by jayvdb** [comment]
:+1: I'll try that

**Comment by jayvdb** [comment]
Adding `publish = false` doesnt help.

**Comment by taiki-e** [comment]
The latest cargo doesn't require package.version field: https://github.com/rust-lang/cargo/pull/12786

However, that does not seem to be included yet in the current stable and will be included in 1.75, which will be released later this month.
